src/Router/Router.tsx

```diff
@@ -12,47 +12,14 @@ export const Router = () => {
     <Routes>
       <Route path="/signin" element={<SignIn />} />
       <Route path="/signup" element={<SignUp />} />
-      <Route path="/" element={<Navigate to="/signin" />} />
-      <Route
-        path="/home"
-        element={
-          <PrivateRoute>
-            <Home />
-          </PrivateRoute>
-        }
-      />
-      <Route
-        path="/profile"
-        element={
-          <PrivateRoute>
-            <Profile />
-          </PrivateRoute>
-        }
-      />
-      <Route
-        path="/test/new"
-        element={
-          <PrivateRoute>
-            <NewTest />
-          </PrivateRoute>
-        }
-      />
-      <Route
-        path="/test/edit/:id"
-        element={
-          <PrivateRoute>
-            <NewTest />
-          </PrivateRoute>
-        }
-      />
-      <Route
-        path="/session/:id"
-        element={
-          <PrivateRoute>
-            <Session />
-          </PrivateRoute>
-        }
-      />
+      <Route path="/" element={<Navigate to="/home" replace />} />
+      <Route element={<PrivateRoute />}>
```

Comment:
To make `PrivateRouter` reusable you can return `children` instead of `Outlet` inside `PrivateRoute` and then you can use it everywhere like this

```suggestion
      <Route element={
        <PrivateRoute>
          <Outlet />
        </PrivateRoute>
      }>
```

or you can use `MainLayout` component instead of `Outlet` if you have one

Reply:
Fully disagreed with your suggestion. Cause PrivateRoute is still fully reusable and also with this changes we recused amount of lines of code. Now we use just one line for any working route and we can just create a layout routes for wrapping other routes together instead of wrapping everyone by it. For the details your you can to try discuss with ME!!!